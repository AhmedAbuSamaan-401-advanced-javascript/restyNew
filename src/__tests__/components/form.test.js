import React from 'react';
import {shallow, mount} from 'enzyme';
import Form from '../../components/form/form';

describe('<Form/>', () => {
  it('its alive at application start', () => {
    const form = shallow(<Form/>);
    expect(form.find('button').exists()).toBeTruthy();
  });
});