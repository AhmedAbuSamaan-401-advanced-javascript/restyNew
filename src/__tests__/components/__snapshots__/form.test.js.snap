// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form/> properly display the users input in the output area on form submit 1`] = `
<main
  className="main"
>
  <div>
    <p
      id="url"
    >
      URL :
    </p>
    <input
      onChange={[Function]}
      type="text"
    />
    <button
      onClick={[Function]}
    >
      GO!
    </button>
  </div>
  <form
    onClick={[Function]}
  >
    <input
      id="get"
      name="method"
      type="radio"
      value="get"
    />
    <label
      for="get"
    >
      Get
    </label>
    <input
      id="post"
      name="method"
      type="radio"
      value="post"
    />
    <label
      for="post"
    >
      Post
    </label>
    <input
      id="put"
      name="method"
      type="radio"
      value="put"
    />
    <label
      for="put"
    >
      Put
    </label>
    <input
      id="delete"
      name="method"
      type="radio"
      value="delete"
    />
    <label
      for="delete"
    >
      Delete
    </label>
  </form>
  <div
    id="output"
  >
    <p>
      
       
      
    </p>
  </div>
</main>
`;
